samples/otlptracegrpc/README.md

```diff
@@ -0,0 +1,37 @@
+# Overview
+
+This example shows how to send traces to an OTLP *(OpenTelemetry Protocol)* endpoint that is protected by GCP authentication over gRPC.
+
+## Installation
+
+```sh
+# from root of repo, build all packages
+npm install
+```
+
+## Run the Application
+
+```sh
+# export necessary OTEL environment variables
+export OTEL_RESOURCE_ATTRIBUTES="gcp.project_id=<project-id>"
+export OTEL_EXPORTER_OTLP_ENDPOINT=<endpoint>
+
+# run the app - this starts app at port 8080
+cd samples/otlptracegrpc && npm run start
+```
+
+## Sending Requests to the Application
+
+In another terminal window:
+
+```sh
+curl localhost:8080/rolldice?rolls=12
```

Comment:
I thoguht this is what the client.ts script was for

Reply:
Client is mostly to assert that the auto-refresh mechanism is working, this shows if you want to do a one-off thing and see the trace being generated.

Reply:
I updated the Readme to reflect the difference b/w the two.